nat-lab/tests/test_lana.py

```diff
@@ -1643,10 +1643,10 @@ async def test_lana_same_meshnet_id_is_reported_after_a_restart(
 @pytest.mark.moose
 @pytest.mark.asyncio
 @pytest.mark.parametrize(
-    "initial_heartbeat_interval", [pytest.param(5), pytest.param(None)]
+    "initial_heartbeat_interval", [pytest.param(5), pytest.param(300)]
```

Comment:
`pytest.param(300)`  is not needed here I think. Previously it was tested that no `initial_heartbeat` happens if it is not configured, so unless `initial_heartbeat_interval = 0` means disabled, other parameter is not needed besides `5`

Reply:
Or maybe nvm the upper comment, it is good to know if we don't create any heartbeats before the actual interval. But would still be nice to check if we get heartbeats if it is disabled (if possible to disable it, like before)

Reply:
It was not possible to disable it before. Even if null was passed the default 300 would be applied in the libtelio. That's why I put 300 here, because it exactly mimics the previous behavior.

Reply:
ah ok, good to know :smile: